test/test.js

```diff
@@ -280,3 +281,32 @@ tape('completion detection when final files visited are not tests', t => {
     t.end();
   });
 });
+
+tape('valid source directory (with hashbang tests)', t => {
+  const fixtureDir = path.join(__dirname, 'collateral', 'valid-with-hashbang');
+  const stream = new TestStream(path.join(fixtureDir, 'fake-test262'));
+  const ids = [];
+
+  stream.on('data', makeDataHandler(t, ids, fixtureDir));
+
+  stream.on('error', (error) => {
+    t.ok(error);
+    t.end(error);
+  });
+
+  stream.on('end', () => {
+    const hashbang = '#!whatever\n';
+    const expectedContentDir = path.join(fixtureDir, 'expected-content');
+    const expectedMetadataDir = path.join(fixtureDir, 'expected-metadata');
+
+    ids.forEach(id => {
+      const startsWithHashbang = fs.readFileSync(
+        path.join(expectedContentDir, `${id}.js`), 'utf8'
+      ).startsWith(hashbang);
+
+      t.equal(startsWithHashbang, true, `${id} begins with "${hashbang}" (the space was a newline character)`);
```

Comment:
The files in the `expected-content` directory are checked in to the repository, and since we're not overwriting them during the test, we know they will always include the hashbang. I think we can remove this assertion without losing coverage.